config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  get 'defects/new'
```

Comment:
Might be good to go more for this style of route definition `resources :comments, only: [:show, :edit, :update, :destroy]` from here http://guides.rubyonrails.org/routing.html